tests/test_cosmologger.py

```diff
@@ -300,3 +300,22 @@ def test_payload_with_dashes(cosmolog, cosmolog_setup):
     assert out['format'] is None
     assert out['payload']['jupiter-ganymede_g'] == 1.428
     assert out['payload']['jupiter-europa_g'] == 1.315
+
+
+def test_user_specified_exc_info(cosmolog, cosmolog_setup):
+    logpath = cosmolog_setup()
+    logger = cosmolog()
+
+    class DeorbitException(Exception):
+        pass
+
+    def fly():
+        raise DeorbitException('Panic. Deorbiting.')
+    try:
+        fly()
+    except DeorbitException as e:
+        logger.exception('Problem flying.', exc_info=True)
```

Comment:
How about adding a test with `exc_info=False` as well?

Reply:
got it.